Datadog/Datadog.xcodeproj/project.pbxproj

```diff
@@ -6432,6 +6432,10 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 61569894256D0E9A00C6AADA /* Base.xcconfig */;
 			buildSettings = {
+				"ARCHS[sdk=iphoneos*]" = (
+					"$(ARCHS_STANDARD)",
+					arm64e,
+				);
```

Comment:
Could we set it through [`Base.xcconfig`](https://github.com/DataDog/dd-sdk-ios/blob/develop/xcconfigs/Base.xcconfig)? It already lists some settings applied to produced `XCFrameworks`, so we could keep it better organised in a single place (`.xcconfig`) instead of duplicating it across all targets (`.pbxproj`).